Add "nodethon" container (Python+Node.js in tox env) (#5)
UAVCAN/docker_toolchains: nodethon/README.md

# "Nodethon" (Node.js + Python) Docker

> Builds and pushes a docker environment for use with tox testing. This environment contains Node.js 13 and some other JavaScript dependencies, besides a series of python versions allowing multi-version tox testing locally and in CI services.

Reviewer: ```suggestion
Builds and pushes a docker environment useful for testing and building Node.js applications that also use Python or Python applications that also use Node.js. This environment contains Node.js and a few other, essential JavaScript dependencies as well as series of python versions allowing multi-version tox testing locally and in CI services.
```

> The "sq" suffix indicates that the ["sonarqube"](https://www.sonarqube.org) scanner has been included in the image. This scanner allows tox builds to upload coverage and other reports to a sonarqube instance.

Reviewer: let's omit the sq suffix for this container. We added that to toxic because it wasn't part of the earlier offering.

> The "sq" suffix indicates that the ["sonarqube"](https://www.sonarqube.org) scanner has been included in the image. This scanner allows tox builds to upload coverage and other reports to a sonarqube instance.

Reviewer: ```suggestion
All nodethon images include the ["sonarqube"](https://www.sonarqube.org) scanner. This scanner allows node and tox builds to upload coverage and analysis reports to a sonarqube instance.
```

## Build and Push

These instructions are for maintainers with permissions to push to the "uavcan" organization on Docker Hub.

```
docker build .
```
```
docker images

REPOSITORY      TAG            IMAGE ID
nodethon        latest         d7ab132649d6
```
```
# We use the range of python environments supported as the version tag.

> docker tag d7ab132649d6 uavcan/nodethon:py35-py38-node13-sq

Reviewer: same comment, search and remove all `-sq` suffixes. Let's also include node version here:

```
uavcan/nodethon:node13-py37-py38
```
